agent/agent.go

```diff
@@ -533,6 +552,12 @@ func main() {
 		log.Errorf("Error while modifying Agent's capabilities: %v", er)
 	}
 
+	er = setupOpenfilesLimit()
+	if er != nil {
+		log.Errorf("Failed to setup open files limit: %v", er)
+		os.Exit(1)
```

Comment:
IMO we should not exit on failure here since, this could be a breaking change for other existing use-cases in which the agent does not have privileges to raise this limit. Since this higher limit is only required in higher load use-cases, we should not block any containers that don't require anything more than the default.

Reply:
Yes. It makes sense to me.